python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -0,0 +1,28 @@
+from ..._core import builtin
+from triton._C.libtriton import ir
+
+__all__ = ["buffer_load_to_shared"]
+
+
+@builtin
+def buffer_load_to_shared(dest, ptr, offsets, mask=None, other=None, stride=None, cache_modifier="", _semantic=None):
+    """
+    AMD Buffer load operation. Similar to amdgpu.buffer_load op but directly writes to shared memory instead of into registers.
```

Comment:
Prefer to clearly define the semantics w/o referencing to other ops to be standalone. You can copy the buffer load description accordingly.

Reply:
This one actually copied from the ops description. I reword it a little bit.